test/index-test.js

```diff
@@ -18,7 +18,7 @@ tests = [{
 }, {
     m: 'a'
   , o: [{b: 1}]
-  , e: null
+  , e: {}
```

Comment:
Why an empty object? I think a `null` is more appropriate since you asked for `a` and that didn't match anything, just like in the second test case.